src/main/groovy/edu/oregonstate/mist/catalogapitest/mapper/InstructorMapper.groovy

```diff
@@ -0,0 +1,25 @@
+package edu.oregonstate.mist.catalogapitest.mapper
+
+import edu.oregonstate.mist.catalogapitest.core.Instructor
+import org.skife.jdbi.v2.StatementContext
+import org.skife.jdbi.v2.tweak.ResultSetMapper
+import java.sql.ResultSet
+import java.sql.SQLException
+
+/**
+ * Mapper for Instructor object properties and database columns.
+ */
+public class InstructorMapper implements ResultSetMapper<Instructor> {
+    public Instructor map(int i, ResultSet rs, StatementContext sc) throws SQLException {
+        Instructor instructor = new Instructor()
+
+        instructor.with {
+            cid               = rs.getInt     'CID'
+            firstInitial      = rs.getString  'FIRST_INITIAL'
+            lastName          = rs.getString  'LAST_NAME'
+            numberOfCourses   = rs.getInt     'NUMBER_OF_COURSES'
+        }
+
+        return instructor
```

Comment:
if you used the named argument constructor, it would not be necessary to define, modify, and return a variable in this and similar methods.